parsec/core/backend_connection/authenticated.py

```diff
@@ -53,6 +54,9 @@
 from parsec.event_bus import EventBus
 from parsec.utils import open_service_nursery
 
+# This global variable activates AuthenticatedCmds binding
+OXIDIZED = False
```

Comment:
When this constant is enabled ? I have the feeling it is always false for the moment
(it's fine to have it disabled in production for the moment, but I guess we'd like to have it enabled for some tests, typically by having a fixture that allow to have the client backend connection related tests to be run with both python and oxidized implementation 🤔 )